cmd/gen-go-sample/out.go

```diff
@@ -75,6 +75,21 @@ func (st *symTab) put(ident string, typ initType) error {
 	return nil
 }
 
+func (st *symTab) disambiguate(ident string, typ initType) string {
```

Comment:
can you add a comment to this function or in the code? I'm having trouble understanding what this code does (might be my lack of context).

Reply:
Done.